### DISCUSSION TITLE
Ceiling light supported?

### DISCUSSION THREAD

**Discussion by GreyEarl** [sbidy/wiz_light#224] [Q&A]
I'm searching for a list of supported Wiz devices. Is the integration literally bulb only or would a Wiz ceiling luminaire works as well?
https://www.wizconnected.com/en/consumer/products/8719514337978/

**Comment by The1Bill** [comment]
I'm using a Designers Fountain 14" overhead light (that works with Wiz) and it works nicely with this integration.
